### ISSUE TITLE
Subdomains Certficates.

### ISSUE THREAD

**Issue by lukealford** [Prajithp/letsencrypt-cpanel#70]
Looks like the plugin doesn't look at sub domains. Can't seem to get my sub-domains in the list from Cpanel.

**Comment by Kurounin** [comment]
It works for me. Check if you already have ssl certificates for those subdomains, in which case you would need to remove them first to be displayed in the list.

**Comment by Robvred** [comment]
Hi, I receive http01 error on all subdomains, but all dns it's fine, do you have an idea ?

**Comment by WilliamIsted** [comment]
@Robvred can you access the files within `.well-known` via a url request?

**Comment by Robvred** [comment]
Yes and by security have add this rule RewriteRule ^.well-known(.*)$ - [L,NC] in .htaccess.
If i go to acme-challenge sub directory i see txt created by let's encrypt, but i receive always this error for all sub domains.
